### DISCUSSION TITLE
Commercial support

### DISCUSSION THREAD

**Discussion by LarsKemmann** [elsa-workflows/elsa-studio#229] [General]
Hello! What this team has done is _amazing_ and I'm planning to incorporate Elsa 3 into a new product architecture that I'm currently working on for a client. However, I haven't been able to find any options anywhere for my client to get commercial support for Elsa 3.

Could you provide some options for a support license? I haven't thought through details, but it would help to have something that allows an enterprise organization to pay for quick turnaround on bugfixes and (perhaps) prioritization of occasional feature requests. Feel free to PM me ([LinkedIn](https://www.linkedin.com/in/lars-kemmann/)) if you'd rather share pricing privately, but it would help to have a public place for information on commercial support options.

**Comment by glime-ai** [comment]
Hello! I'm glad to hear about your interest in incorporating Elsa 3 into your new product architecture. While I can provide you with general information based on the context provided, for specific details regarding commercial support options, it would be best to directly reach out to the Elsa Workflow project maintainers.

As of the information available to me, there is no explicit mention of commercial support licenses or services in the provided context. However, since Elsa is an open-source project, you can typically get community support through various channels such as GitHub issues, Stack Overflow, or the project's Discord server. For enterprise-level support, it's common to engage with the core maintainers or contributors on a contractual basis.

I recommend reaching out to the project maintainers directly for inquiries about commercial support. You can do this by creating an issue on the Elsa Workflow GitHub repository or by contacting the maintainers through the contact details they might have provided in the repository or on their personal GitHub profiles.

If you prefer, you can also try to connect with them on LinkedIn or any other professional network where they are active. Since you've mentioned LinkedIn, you might want to search for the profiles of the core contributors or maintainers of Elsa Workflows and reach out to them with your proposal.

Remember to explain your needs clearly, such as the requirement for quick turnaround on bug fixes and feature request prioritization, as this will help in discussing a tailored support plan that suits your enterprise needs.

This comment was generated by [Glime](https://glimelab.ai/?utm_source=dRbvTb7IhCSYiI&utm_medium=github).
